concepticondata/conceptrelations.tsv

```diff
@@ -1217,3 +1217,19 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 3074	FIX	instanceof	2243	REPAIR
 1755	UNDRESS	instanceof	3538	TAKE OFF
 158	PUT ON	antonymof	3538	TAKE OFF
+3603	EARTH OR LAND	instanceof	1228	LAND
+3603	EARTH OR LAND	instanceof	626	LAND
```

Comment:
this is not possible: I assume 626 and 1228 have different concepticon glosses, right?

Reply:
Or you should write "narrower"

Reply:
I suspect even if "EARTH OR LAND" needs to be linked here: it seems that they rather mean the land as in concepticon LAND. 
And of course, the concept needs to link to SOIl then as well (narrower relation).

Reply:
> this is not possible: I assume 626 and 1228 have different concepticon glosses, right?

My mistake, will fix.